pkg/config/config.go

```diff
@@ -46,12 +46,13 @@ type Stage struct {
 }
 
 type StageParam struct {
-	Name      string     `json:"name"`
-	Ingest    *Ingest    `json:"ingest,omitempty"`
-	Transform *Transform `json:"transform,omitempty"`
-	Extract   *Extract   `json:"extract,omitempty"`
-	Encode    *Encode    `json:"encode,omitempty"`
-	Write     *Write     `json:"write,omitempty"`
+	Name      string         `json:"name"`
+	Ingest    *Ingest        `json:"ingest,omitempty"`
+	Transform *Transform     `json:"transform,omitempty"`
+	ConnTrack *api.ConnTrack `json:"conntrack,omitempty"`
```

Comment:
Should we consider something more generic for consistency like `track` ?

Reply:
The inconsistency bothers me too. At first I added an abstraction layer, but then I removed it because the config ended up with consecutive levels of `conntrack` which looked funny to me:
```yaml
pipeline: ...
parameters: ...
  - name: conntrack
    contrack:
      conntrack:
        ...
      type: ...
```
https://github.com/netobserv/flowlogs-pipeline/pull/247/commits/30aa7e8382f13e1c422a35581af2efb63cbbb1ec#diff-81d2b1554fae3c0d5186eeb17c96d0dbb9373fd8867220245e2612b4bd26849eL45-L50

I didn't think of making the added layer a generic layer for tracking. I'll take your advice :)

Reply:
yeah let's keep it simple :smile:

Reply:
but conn_track couldn't be an implementation of a standard Transform stage ?

Reply:
I'll follow @KalmanMeth and your suggestion and make `conntrack` a type of `extract`